relay-server/src/endpoints/common.rs

```diff
@@ -369,6 +369,10 @@ where
             metric!(counter(RelayCounters::EnvelopeRejected) += 1);
             let event_id = *event_id.borrow();
 
+            if Capture::should_capture(&config) {
+                EnvelopeManager::from_registry().do_send(Capture::rejected(event_id, &error));
+            }
```

Comment:
Was this case previously uncaptured? Or did it happen somewhere else?

Reply:
This was previously not captured, correct. The fast path also bypassed other means of the envelope manager, such as the buffer capacity check.